core/src/accounts_hash_verifier.rs

```diff
@@ -155,17 +179,53 @@ impl AccountsHashVerifier {
             }
         }
 
+        cluster_info.push_accounts_hashes(hashes.clone());
+    }
+
+    fn handle_snapshot(
```

Comment:
nit: maybe rename to `queue_snapshot_for_packaging()`

Reply:
Done!